analytics/filesystem/file_module_test.go

```diff
@@ -23,6 +24,8 @@ func TestAmpObject_ToJson(t *testing.T) {
 	}
 	if aoJson := jsonifyAmpObject(ao); strings.Contains(aoJson, "Transactional Logs Error") {
 		t.Fatalf("AmpObject failed to convert to json")
+	} else {
+		fmt.Println(aoJson)
```

Comment:
For my own understanding, why do we need a print statement in a test file?

Reply:
We don't. I accidentally left this in from manual exploration.